app/views/course/assessment/answer/programming/_programming.html.slim

```diff
@@ -10,3 +10,5 @@ div.files
 - answer = last_attempt ? last_attempt.actable: programming
 = render partial: 'course/assessment/answer/programming/test_cases',
          locals: { question: question, answer: answer }
+= render partial: 'course/assessment/answer/programming/output_streams',
```

Comment:
Shall we move the checks from the partial into here instead? So we don't render the partial unless required.

```ruby
- auto_grading = answer.auto_grading ? answer.auto_grading.actable : nil
- is_grader = can?(:read_tests, @submission)
```